test/src/test/java/test/org/dst/client/SetProxyTest.java

```diff
@@ -0,0 +1,86 @@
+package test.org.dst.client;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.dst.client.DefaultDstClient;
+import org.dst.exception.DstException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import test.org.dst.supplier.BaseTestSupplier;
+
+public class SetProxyTest extends BaseTestSupplier {
+
+  private final static String serverAddress = "list://127.0.0.1:8082";
+
+  @Test
+  public void testPutAndGet() {
+    Set<String> set = new HashSet<>();
```

Comment:
```Java
Set<String> set = ImmutableSet.of("v1", "v2");

```

Reply:
I just want to test the property of the set. The element of set should be different.

Reply:
Done.